Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/RaceResultsListAdapter.java

```diff
@@ -122,8 +122,9 @@ public View getGroupView(int groupPosition, boolean isExpanded, View convertView
     private String createGroupViewText (int position, int laps, boolean isFinished) {
         String ch = AppState.getInstance().getChannelText(position);
         String band = AppState.getInstance().getBandText(position);
+        String freq = AppState.getInstance().getFrequencyText(position);
         DeviceState ds = AppState.getInstance().deviceStates.get(position);
-        return ds.pilotName + " (C " + ch + ", " + band + " band) Laps: " + (laps <= 0 ? "-" : laps) + (isFinished ? " FINISHED" : "");
+        return ds.pilotName + " (C " + ch + ", " + band + " band, " + freq +") Laps: " + (laps <= 0 ? "-" : laps) + (isFinished ? " FINISHED" : "");

```

Comment:
Let's put frequency after the band, please (I think it will be a bit more readable)